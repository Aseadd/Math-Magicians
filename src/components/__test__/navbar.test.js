import React from 'react';
import ReactDOM from 'react-dom/client';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Navbar from '../navbar';

it('should render the navbar component', () => {
  const wrapper = ReactDOM.createRoot(document.createElement('div'));
  wrapper.render(<Navbar />);
});

it('should match the snapshot', () => {
  const tree = renderer
    .create(
      <Router>
        <Navbar />
      </Router>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
