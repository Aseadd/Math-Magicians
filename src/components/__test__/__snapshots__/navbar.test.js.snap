// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<div>
  <nav
    className="pt-3 pb-3 nav-bar "
  >
    <ul
      className="nav nav-pills justify-content-end pe-5 me-3"
    >
      <li
        className="nav-item me-auto"
      >
        <a
          aria-current="page"
          className="nav-link text-white"
          href="/"
        >
          Math Magician
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          aria-current="page"
          className="nav-link text-white active"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link text-white"
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link text-white"
          href="/quotes"
          onClick={[Function]}
        >
          Quotes
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
